projects/github-actions/repo-gardening/src/tasks/add-labels/index.js

```diff
@@ -184,8 +184,8 @@ async function getLabelsToAdd( octokit, owner, repo, number, isDraft, isRevert )
 			keywords.add( '[Tools] Development CLI' );
 		}
 
-		const docs = file.match( /^docs\/|\.md$/ ) && ! file.match( /\/CHANGELOG\.md$/ );
-		if ( docs !== null ) {
+		const docs = file.match( /^docs\/|\.md$/ ) && ! file.match( /\/?CHANGELOG\.md$/i );
```

Comment:
If you're intending to ignore files like `foobarchangelog.md`, you could simplify it to this
```suggestion
		const docs = file.match( /^docs\/|\.md$/ ) && ! file.match( /CHANGELOG\.md$/i );
```
If you only want to ignore a root-level `changelog.md` but not `foobarchangelog.md`, do like this instead to match "start-of-string or /"
```suggestion
		const docs = file.match( /^docs\/|\.md$/ ) && ! file.match( /(?:^|\/)CHANGELOG\.md$/i );
```